build.gradle

```diff
@@ -450,7 +450,6 @@ tasks.withType(ShadowJar) {
     mergeServiceFiles()
     relocate 'com.google.common', 'org.broadinstitute.hellbender.relocated.com.google.common'
     zip64 true
-    exclude 'log4j.properties' // from adam jar as it clashes with hellbender's log4j2.xml
```

Comment:
isn't this going to reintroduce the clash this exclusion sought to avoid?

Reply:
Yes - I'm trying to find out what the nature of that clash was.